src/ids-badge/ids-badge.js

```diff
@@ -0,0 +1,96 @@
+import {
+  IdsElement,
+  customElement,
+  scss
+} from '../ids-base/ids-element';
+// @ts-ignore
+import styles from './ids-badge.scss';
+
+/**
+ * IDS Badge Component
+ */
+@customElement('ids-badge')
+@scss(styles)
+class IdsBadge extends IdsElement {
+  constructor() {
+    super();
+  }
+
+  /**
+   * Return the properties we handle as getters/setters
+   * @returns {Array} The properties in an array
+   */
+  static get properties() {
+    return ['color', 'shape'];
```

Comment:
```suggestion
    return ['color', 'shape'];
```

Could put this in ids-constants (color exists already)